nip57.ts

```diff
@@ -1,4 +1,5 @@
 import { bech32 } from '@scure/base'
+const bolt11 = require('light-bolt11-decoder')
```

Comment:
Why is this a require instead of an ES import?

Reply:
Because there's no types for it.